### PR TITLE
Add: Test case for using a list with select_class in common bundles

### PR DESCRIPTION
**Pull request by nickanderson**
Ref: https://dev.cfengine.com/issues/7482

### CODE REVIEW

**Comment by nickanderson** [comment]
Note this has to do with how pre-eval works. More details in the ticket: https://dev.cfengine.com/issues/7482

File: tests/acceptance/02_classes/01_basic/common_bundle_select_class_list.cf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      "common_classes" slist => { "common_1", "common_2" };
+
+    common_class_selected::	
+      "class_selected"
+        string => "$(common_classes)",
+        if => "$(common_classes)";
+  
+  classes:
+      # During pre-eval classes are checked first. During that initial pass the
+      # common_classes list is undefined, and so the promise should be skipped.
```
Comment (pasinskim) [inline tests/acceptance/02_classes/01_basic/common_bundle_select_class_list.cf]: This is not quite true unfortunately. It won't be evaluated during first pre-evaluation round but there are multiple and should be evaluated in subsequent.

Comment (nickanderson) [inline tests/acceptance/02_classes/01_basic/common_bundle_select_class_list.cf]: So you want me to adjust to "during the first pre-eval pass classes are checked first"?

Or are you saying the test is invalid?

**Comment by kacfengine** [comment]
Covered in https://github.com/cfengine/core/pull/2327.

**Pull request closed without merging**